### PR TITLE
Set default value when null is passed to GraphQL API

### PR DESCRIPTION
**Pull request by jacekwegr**
This PR changes how arguments are processed before being passed to the resolver. When an argument has an explicit null value, the GraphQL library tries to set its value to a default one if one exists.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/143240) / small_tests / 3b1c40d3481152eec80df6fee0e5fac641bf7985
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3803/143240/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3803/143240/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-10-10_11.35.25/lib.mongooseim.logs/run.2022-10-10_11.35.26/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/145050) / small_tests / a8bde9d9261f5298a21ac53ca4908453e6bfee06
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3803/145050/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3803/145050/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-10-17_07.47.23/lib.mongooseim.logs/run.2022-10-17_07.47.24/suite.log.html)

File: rebar.config [hunk 1]
```diff
@@ -77,7 +77,7 @@
   {worker_pool, "6.0.1"},
 
   %%% HTTP tools
-  {graphql, {git, "https://github.com/esl/graphql-erlang.git", {branch, "master"}}},
+  {graphql, {git, "https://github.com/esl/graphql-erlang.git", {branch, "set-default-when-null-is-passed-explicitly"}}},
```
Comment (JanuszJakubiec) [inline rebar.config:80]: this should be changed back to master after library is merged

**Review by JanuszJakubiec — APPROVED** [review]

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/145161) / small_tests / def8dce71554bae3a026ca2d0616c813ff2faa94
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3803/145161/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3803/145161/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-10-18_09.59.37/lib.mongooseim.logs/run.2022-10-18_09.59.38/suite.log.html)

**Pull request merged**